requirements.txt

```diff
@@ -15,9 +15,10 @@ jedi>=0.18.1,<0.19
 matplotlib==3.4.3
 multiprocess==0.70.13
 natsort>=8.1.0,<9
-numpy>=1.21.6,<2
+numpy>=1.23.4,<2
 palettable>=3.3.0,<4
 pandas>=1.3.5,<2
+pillow>=9.1.0,<9.2
```

Comment:
We won't be able to merge this in until pyFlowSOM is in, since this would require rebuilding the docker.